lib/rotoscope.rb

```diff
@@ -34,17 +34,16 @@ def self.trace(dest, blacklist: [], flatten: false)
   end
 
   def flatten(dest)
-    io_given = false
-    dest_file = if dest.respond_to?(:to_io)
-      io_given = true
-      dest.to_io
+    io_given = if dest.respond_to?(:puts)
```

Comment:
I don't know of a better way to check for support (since things like `Zlib::GzipWriter` don't actually have any `IO` in their ancestor tree), but this is the only method invoked on the object, so it seems sane from my perspective